src/libraries/System.Linq.Expressions/tests/InterpreterTests.cs

```diff
@@ -15,7 +11,9 @@ public static class InterpreterTests
     {
         private static readonly PropertyInfo s_debugView = typeof(LightLambda).GetPropertyAssert("DebugView");
 
-        [Fact]
+        // FEATURE_COMPILE is not directly required,
```

Comment:
>FEATURE_COMPILE [](http://example.com/codeflow?start=11&length=15)

Should the comment say `LambdaExpression.CanCompileToIL` instead, here and below, since there are no longer references to `FEATURE_COMPILE`?

Reply:
Fixed those all.

Once this is merged, I'll do a test pass on CoreCLR with the IL Compiler disabled and see whether these comments are correct. If the comment is correct, I'll change the condition from "is IL compiler available" to "is Reflection.Emit available" and delete the comment, because that's what we're after.